MAINTAINERS

```diff
@@ -15,3 +15,4 @@
 "jsturtevant","James Sturtevant","jstur@microsoft.com"
 "yuchanns", "Hanchin Hsieh", "me@yuchanns.xyz"
 "manugupt1", "Manu Gupta", "manugupt1@gmail.com"
+"djdongjin", "Jin Dong", "jindon@amazon.com"
```

Comment:
> `jindon`

Might be a typo of `jindong`?

Reply:
![image](https://user-images.githubusercontent.com/7054676/207401653-77b64b64-7898-492f-8b76-e6af9761baff.png)
 
No, seems he used this as his git email

Reply:
It's actually `jindon` 😅 I'm not lucky to get my full name as my alias 😅